go.mod

```diff
@@ -47,6 +47,7 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/fluentforwardreceiver v0.40.1-0.20211206165607-024958a4f3c4
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver v0.40.1-0.20211206165607-024958a4f3c4
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jaegerreceiver v0.40.1-0.20211206165607-024958a4f3c4
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/journaldreceiver v0.40.0
```

Comment:
Please upgrade to `v0.40.1-0.20211206165607-024958a4f3c4` version

Reply:
Thanks the for the correction.